docs/source/markdown/podman-pause.1.md.in

```diff
@@ -10,6 +10,7 @@ podman\-pause - Pause one or more containers
 
 ## DESCRIPTION
 Pauses all the processes in one or more containers.  You may use container IDs or names as input.
+If a HealthCheck interval is configured for the container, the timer for that interval is removed.
```

Comment:
An average user has likely no idea what that means. They don't know how healthchecks are run or this would be relevant

I don't see the value in this sentence because it just describes implementation details. If we want to mention the case it should just say healthchecks are stopped.